packages/yourloops/components/dialogs/remote-monitoring-dialog.tsx

```diff
@@ -95,32 +105,59 @@ function RemoteMonitoringPatientInviteDialog(props: RemoteMonitoringPatientInvit
   const [saveButtonDisabled, setSaveButtonDisabled] = useState(true);
 
   const onSave = async () => {
-    const monitoringEnd = moment.utc(new Date()).add(prescriptionInfo.numberOfMonth, "M").toDate();
-    if (!prescriptionInfo.teamId) {
-      throw Error("Cannot invite patient as remote monitoring team id has not been defined");
-    }
-    if (!prescriptionInfo.memberId) {
-      throw Error("Cannot invite patient as prescriptor id has not been defined");
-    }
-    if (!prescriptionInfo.file) {
-      throw Error("Cannot invite patient as prescription has not been defined");
+    try {
+      setSaveButtonDisabled(true);
+      const monitoringEnd = moment.utc(new Date()).add(prescriptionInfo.numberOfMonth, "M").toDate();
+      if (!prescriptionInfo.teamId) {
+        throw Error("Cannot invite patient as remote monitoring team id has not been defined");
+      }
+      if (!prescriptionInfo.memberId) {
+        throw Error("Cannot invite patient as prescriptor id has not been defined");
+      }
+      if (!prescriptionInfo.file) {
+        // missing trad...
```

Comment:
Remove this comment?

Reply:
benn this is a a todo for late, to not forget , Do you prefer to have a Jira for this ?

Reply:
Yes please at least create a JIRA